### ISSUE TITLE
Page nesting

### ISSUE THREAD

**Issue by grantmcdermott** [LOST-STATS/lost-stats.github.io#55]
See: https://github.com/LOST-STATS/lost-stats.github.io/issues/5#issuecomment-674158878

We want to tidy up page hierarchies, starting with **Model Estimation** and maybe move on to some of the other sections. The tricky thing is deciding the best categories, although some overlap is probably unavoidable. Here's a first stab... mostly working off the existing pages, but with one or two non-existing (but obvious) counterparts thrown in. Feel to edit or make changes.

- Ordinary Least Squares
   - (Simple?) Linear Regression
   - Fixed Effects 
   - Interaction terms and Polys
- Generalised Least Squares
   - Logit
   - Probit
   - Tobit
   - Heckman
   - McFadden / clogit
- Multilevel (hierarchical?) models
   - Random effects
   -  Linear mixed effects
   - Bayesian hierarchical models
   - etc
- Inference (maybe "Design"?)
   - DiD
   - RDD
   - IV
   - Linear Hypothesis Tests
- Nonstandard errors 
   - *This sub-section is already nested*

**Comment by NickCH-K** [comment]
I like this setup! I think GLS will likely get big enough at some point that it will need to be split further, but its not there yet. 

Maybe take Linear Hypothesis Tests and put it in a section called "Statistical Inference" that would also include standard errors?

**Comment by grantmcdermott** [comment]
Hmm... I like the 'statistical inference' section header. I'll use 'research design' for the DiD stuff then to avoid confusion between section names.

**Comment by NickCH-K** [comment]
Sounds good to me

**Comment by grantmcdermott** [comment]
Okay @NickCH-K it's pretty much done (for model estimation at least). 

https://lost-stats.github.io/Model_Estimation/Model_Estimation.html

*Minor note:* Since this is going to result in a bunch of broken internal hyperlinks -- still needs to be fixed -- I went ahead and also changed the underlying Estimation.md file to Model_Estimation.md to improve file naming <--> navigation consistency.

**Comment by grantmcdermott** [comment]
Oh, another question: 

Shouldn't Time Series (and maybe even Machine Learning) be nested under Model Estimation?

**Comment by grantmcdermott** [comment]
If I'm picky, we should probably split the 'Presentation' section into 'Figures' and 'Tables' subsections and move 'Summary Statistics' into the latter...

**Comment by NickCH-K** [comment]
Figures and Tables separate makes sense.

If Model Estimation gets all of Time Series and ML that's going to make it huge. Maybe Model Estimation gets a title that excludes those two somehow? I'm not sure what that would be.

**Comment by khwilson** [comment]
A note on the broken links: Would you like me to install `htmlproofer`? It can run on build and let you know what the broken links are.

**Comment by NickCH-K** [comment]
That would be awesome! It was a pain trawling through to find all the broken links the last time we did something like this, and the site was smaller then.

Nick Huntington-Klein
Assistant Professor of Economics
Seattle University

**Comment by khwilson** [comment]
Cross referencing #59 which adds in the proofer. Note that it requires switching the linking style as currently internal links are being implemented as absolute links (i.e., `https://lost-stats.github.io/Time_Series` versus just `/Time_Series`). There's a bit of a cleanup project to switch those, but afterwards, we can, if you'd like, make the GH Actions job fail if there are missing links.

(Note that this may not be desirable as, e.g., the Time Series section currently links to, say, ARIMA models even though the page doesn't exist. OTOH, perhaps the requirement for merge would all those pages get placeholders?)

**Comment by grantmcdermott** [comment]
Awesome @khwilson. On your follow-up points:

1. The use of absolute paths has been bothering me for a while, so I'm happy to see those go.
2. Getting the scaffolding up for the desired non-existent pages would make it easier for contributors to add code and ensure nesting stays correct. (Just edit the existing, blank template.) But I'll be honest and sayt hat I don't have time rn to dedicate to that!

**Comment by NickCH-K** [comment]
1. Agreed!
2. I don't have time for this either, but this would be a good solution (I've long wanted the default missing-page result to be a Wikipedia - style "write your own!" but I imagine this isn't possible. I don't have time for this now but could do it in September.

On Aug 17, 2020 10:19 AM, Grant McDermott <notifications@github.com> wrote:

Awesome @khwilson<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkhwilson&data=02%7C01%7Cnhuntington-klein%40seattleu.edu%7C96a586250695404d829e08d842d1b386%7Cbc10e052b01c48499967ee7ec74fc9d8%7C0%7C0%7C637332815722918454&sdata=l1pB5N7mPD2FqJ2AYH8J%2B4%2BC9PCS7Yu5WbflDOE4oAI%3D&reserved=0>. On your follow-up points:

  1.  The use of absolute paths has been bothering me for a while, so I'm happy to see those go.
  2.  Getting the scaffolding up for the desired non-existent pages would make it easier for contributors to add code and ensure nesting stays correct. (Just edit the existing, blank template.) But I'll be honest and sayt hat I don't have time rn to dedicate to that!

**Comment by khwilson** [comment]
1. Sounds good. We hopefully have a new season of interns arriving sometime in the fall, so there's some chance I can make this an intern project. Just have to haggle with the intern czar. :-)

2. You actually can create a custom 404 page in gh pages, GH serves the 404 from the address you linked to. See e.g. [this nonexistent page](https://lost-stats.github.io/asdf.html). So a little JavaScript could probably do exactly what you're asking Nick.

2b. If you setup the TOC as a YAML, it's very easy to write a tiny script to just create all the blank pages and wouldn't take too long.

**Comment by grantmcdermott** [comment]
Okay, I've nested the Presentation section now too: https://lost-stats.github.io/Presentation/Presentation.html

FWIW I fixed the (absolute) paths here with a [quick bit](https://github.com/LOST-STATS/lost-stats.github.io/commit/7582c1dec59764a8b2f3b405668c833fb1d351ac) of `sed` work since otherwise none of the figures would render. I probably should do the same for the Model Estimation section... but maybe that should wait seeing as Kevin is setting up the HTML checker anyway? Maybe I'll see if I have time later.

I'm going to close this issue now since the main nesting cases have been taken care of.
